### DISCUSSION TITLE
Possibility to query 2 different indexes with different filters

### DISCUSSION THREAD

**Discussion by kovalska854** [jertel/elastalert2#1207] [Q&A] [answered]
Hi there,

Is there any way to query 2 different indexes with different filters? 
My idea is to get results from different types of data and compare it for needed matches in enhancement (as an example). The same feature was implemented in watchers (see [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/input-chain.html)) 
If there is no prebuilt function in elastalert2, can you tell me which function can be used to manually query an elastic for a second time?

**Comment by jertel** [comment, marked as answer]
There is no capability to perform multiple queries on a single rule run. There is the possibility to use a single query against multiple filters, though, provided the `index:` value is using a wildcard pattern that covers both indices. You can also specify the `_index: my-index` field/value in the query, along with AND/OR to combine query clauses.

When in the enhancement process method, you could access the ES client via the util.py::elasticsearch_client(conf) method.

**Reply by kovalska854** [reply]
Got it, many thanks for your respond
